gflownet/envs/base.py

```diff
@@ -36,8 +36,11 @@ def __init__(
         proxy_state_format: str = "oracle",
         fixed_distribution: Optional[dict] = None,
         random_distribution: Optional[dict] = None,
+        conditional=False,
```

Comment:
We should also add this variable to the config, so that it can be controlled via config or command line. I will do it.